package.json

```diff
@@ -1,78 +1,93 @@
 {
   "name": "sanity-plugin-icon-picker",
```

Comment:
I updated the version here, though the original author may want to do it himself